stm32-lvgl/.sourcekit-lsp/config.json

```
@@ -0,0 +1,14 @@
+{
+  "swiftPM": {
+    "configuration": "release",
+    "triple": "armv7em-none-none-eabi",
+
+    "__comment": "XXX SourceKit-LSP seems to ignore the toolset (relative or absolute path)...",
+    "toolset": ".../toolset.json",
```

This is probably meant to be something like this:
```suggestion
    "toolsets": ["../toolset.json"],
```

---

Also, this fix is required too.

https://github.com/swiftlang/sourcekit-lsp/issues/2148

---

fixed!